include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -410,17 +410,22 @@ def TTNN_MaxOp : TTNN_ReductionOp<"max"> {
   }];
 }
 
-def TTNN_EmbeddingOp : TTNN_Op<"embedding"> {
+def TTNN_EmbeddingOp : TTNN_NamedDPSOp<"embedding"> {
```

Comment:
I belive @jnie-TT covers this in this PR:
https://github.com/tenstorrent/tt-mlir/pull/1128/files